tests/integration/staking/keeper/determinstic_test.go

```diff
@@ -36,45 +30,6 @@ var (
 	delegatorAddr2    = sdk.MustAccAddressFromBech32(delegator2)
 )
 
-type DeterministicTestSuite struct {
-	suite.Suite
-
-	ctx           sdk.Context
-	stakingKeeper *stakingkeeper.Keeper
-	bankKeeper    bankkeeper.BaseKeeper
-	accountKeeper authkeeper.AccountKeeper
-	queryClient   stakingtypes.QueryClient
-	amt1          math.Int
-	amt2          math.Int
-}
-
-func (s *DeterministicTestSuite) SetupTest() {
```

Comment:
This shouldn't be removed, for deterministic tests we should use independent `fixture`.